SlackReactor.cs

```diff
@@ -17,6 +18,13 @@ namespace Seq.Slack
     [SeqApp("Slack Notifier", Description = "Sends messages matching a view to Slack.")]
     public class SlackReactor : Reactor, ISubscribeTo<LogEventData>
     {
+        static Regex placeholdersRx = new Regex("(\\[(?<key>[^\\[\\]]+?)(\\:(?<format>[^\\[\\]]+?))?\\])", RegexOptions.CultureInvariant | RegexOptions.Compiled);
+
+        [SeqAppSetting(
+            DisplayName = "Seq Base URL",
+            HelpText = "Used for generating perma links to events in Slack messages.",
+        IsOptional = true)]
+        public string BaseUrl { get; set; }
```

Comment:
If the Seq server is configured with `--listen="http://some/url` this will already be available in `Host.ListenUris` from the reactor class. Still handy to have as a setting here - just an FYI :-)